modules/virt-runbook-virtualmachinecrcerrors.adoc

```diff
@@ -27,41 +27,45 @@ severe performance degradation.
 [id="diagnosis-virtualmachinecrcerrors"]
 == Diagnosis
 
-. Get the volume name from a virtual machine:
+. Navigate to *Observe* -> *Metrics* in the web console.
+. Obtain a list of VirtualMachines with incorrectly configured storage classes
+by running the following PromQL query:
```

Comment:
Join to a single line.

Edit: Also the handful of other instances like this in the PR, where the sentence is broken across multiple lines.

Reply:
Unavoidable. These files are generated from Markdown, in a repo with a linter that requires these line breaks.